build.pkr.hcl

```diff
@@ -11,11 +11,23 @@ build {
       "./files/graphdb-cluster-proxy.service",
       "./files/install_graphdb.sh",
       "./files/cloudwatch-agent-config.json",
-      "./files/prometheus.yaml"
+      "./files/prometheus.yaml",
+      "./files/create_backup.sh",
+      "./files/ebs_volume.sh",
+      "./files/register_route53.sh"
     ]
     destination = "/tmp/"
   }
 
+  provisioner "file" {
+    sources = [
+      "./files/ebs_volume.sh",
```

Comment:
why do you duplicate the files 
You have them once in `tmp` and then again in `/opt/helper-scripts`?
Furthermore these file would have root permissions as is, the `graphdb` user won't be able to execute them.